kola/tests/sysext/nvidia.go

```diff
@@ -0,0 +1,283 @@
+package sysext
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/coreos/pkg/capnslog"
+	"github.com/flatcar/mantle/kola"
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+	testsutil "github.com/flatcar/mantle/kola/tests/util"
+	"github.com/flatcar/mantle/platform"
+	"github.com/flatcar/mantle/platform/conf"
+	"github.com/flatcar/mantle/util"
+)
+
+const (
+	CmdTimeout           = time.Second * 300
+	NvidiaDriversVersion = "550-open"                         // NVIDIA drivers sysext version used in the template
+	KubernetesVersion    = "v1.30.8"                          // Kubernetes version used in the template
+	NvidiaRuntimeVersion = "v1.16.2"                          // NVIDIA runtime version used in the template
+	GpuOperatorVersion   = "v24.9.2"                          // GPU operator version used for Helm install
+	CudaSampleImageTag   = "vectoradd-cuda11.7.1-ubuntu20.04" // CUDA sample image tag
+)
+
+const nvidiaTemplate = `
+variant: flatcar
+version: 1.0.0
+
+storage:
+  files:
+  - path: /opt/extensions/flatcar-nvidia-drivers/flatcar-nvidia-drivers-{{ .NvidiaDriversVersion }}.raw
+    mode: 0644
+    contents:
+      source: https://bincache.flatcar-linux.net/images/amd64/9999.9.4+nvidia-drivers-signing/flatcar-nvidia-drivers-{{ .NvidiaDriversVersion }}.raw
+  links:
+  - path: /etc/extensions/flatcar-nvidia-drivers-{{ .NvidiaDriversVersion }}.raw
+    target: /opt/extensions/flatcar-nvidia-drivers/flatcar-nvidia-drivers-{{ .NvidiaDriversVersion }}.raw
+    hard: false
```

Comment:
For "official" sysexts, the way to enable them is to write to `/etc/flatcar/enabled-sysext.conf`.

Could you rework the test to use that method?

Docs: https://www.flatcar.org/docs/latest/provisioning/sysext/#types-of-systemd-sysext-images
Example in the zfs test: https://github.com/flatcar/mantle/blob/fb1dd2fdfdab1a4c2ea589c64bbf5659417449b7/kola/tests/sysext/zfs.go#L94-L97.

Reply:
Fixed now